### ISSUE TITLE
Environment Error

### ISSUE THREAD

**Issue by wuyun-long** [zetane/viewer#13]
When I load a keras file in .h5 format, an environment error is reported. Why?

**Comment by konafah** [comment]
Hello. Thanks for trying Zetane. Please try loading the h5 again, then closing the viewer, then sending us the latest `zetane_error_log` .

The logs can be found in:
* macOS: `~/Library/Application\ Support/Zetane/logs`
* Ubuntu: `~/.config/Zetane/logs`
* Windows: `%APPDATA%/Zetane/logs`

You can also zip all the logs and email me directly at omar@zetane.com

**Comment by wuyun-long** [comment]
Thank you for your reply. In the attachment is the log file of the program, currently I encountered two kinds of problems 1. environment error, 2. value error. Thank you for your reply again!
Yours sincerely,
Wuyun-long





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zetane/viewer"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月18日(星期五) 晚上10:27
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [zetane/viewer] Environment Error (Issue #13)





 
Hello. Thanks for trying Zetane. Please try loading the h5 again, then closing the viewer, then sending us the latest zetane_error_log .
 
The logs can be found in:
  
macOS: ~/Library/Application\ Support/Zetane/logs
 
Ubuntu: ~/.config/Zetane/logs
 
Windows: %APPDATA%/Zetane/logs
  
You can also zip all the logs and email me directly at ***@***.***
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by konafah** [comment]
Thank you for the logs, here's what we saw:

1. For the first error, it looks like pypi (files.pythonhosted.org) was not reachable from your network, so the keras environment setup was not able to complete. However, in later attempts (02-21-2022) it looks like the connectivity issue was resolved and the environment completed its setup correctly.

2. The second error `ValueError: No model config found in the file` can happen when the h5 file only has weights. Try re-saving the whole model (configuration + weights) as suggested here:
    - [Keras FAQ: What are my options for saving models?](https://keras.io/getting_started/faq/#what-are-my-options-for-saving-models)
    - https://github.com/keras-team/keras/issues/6937.
